tests/testBurrow.ml

```diff
@@ -741,6 +889,102 @@ let suite =
             "burrow_is_cancellation_warranted returned false, but the cancellation is expected to be warranted"
             (Burrow.burrow_is_cancellation_warranted Parameters.initial_parameters burrow cancelled_slice_tez))
     );
+
+    (
+      "burrow_is_cancellation_warranted - unwarranted cancellation" >::
+      fun _ ->
+        let burrow = Burrow.make_burrow_for_test
+            ~outstanding_kit:(kit_of_mukit (Ligo.nat_from_literal "4_657_143n"))
+            ~excess_kit:kit_zero
+            ~active:true
+            ~address:alice_addr
+            ~delegate:None
+            ~collateral:(Ligo.tez_from_literal "5_000_000mutez")
+            ~adjustment_index:fixedpoint_one
+            ~collateral_at_auction:(Ligo.tez_from_literal "3_000_000mutez")
+            ~last_touched:(Ligo.timestamp_from_seconds_literal 0) in
+        let cancelled_slice_tez = Ligo.tez_from_literal "1_000_000mutez" in
+
+        assert_bool
+          "burrow_is_cancellation_warranted returned true, but the cancellation is expected to be unwarranted"
+          (not (Burrow.burrow_is_cancellation_warranted Parameters.initial_parameters burrow cancelled_slice_tez))
+    );
+
+    (
+      "burrow_is_liquidatable - liquidatable burrow" >::
+      fun _ ->
+        let burrow = Burrow.make_burrow_for_test
+            ~outstanding_kit:(kit_of_mukit (Ligo.nat_from_literal "4_000_000n"))
+            ~excess_kit:kit_zero
+            ~active:true
+            ~address:alice_addr
+            ~delegate:None
+            ~collateral:(Ligo.tez_from_literal "2_469_999mutez")
+            ~adjustment_index:fixedpoint_one
+            ~collateral_at_auction:(Ligo.tez_from_literal "3_000_000mutez")
+            ~last_touched:(Ligo.timestamp_from_seconds_literal 0) in
+
+        assert_bool
+          "burrow_is_liquidatable returned false, but the burrow is expected to be liquidatable"
+          (Burrow.burrow_is_liquidatable Parameters.initial_parameters burrow)
+    );
+
+    (
+      "burrow_is_liquidatable - non-liquidatable burrow" >::
+      fun _ ->
+        let burrow = Burrow.make_burrow_for_test
+            ~outstanding_kit:(kit_of_mukit (Ligo.nat_from_literal "4_000_000n"))
+            ~excess_kit:kit_zero
+            ~active:true
+            ~address:alice_addr
+            ~delegate:None
+            ~collateral:(Ligo.tez_from_literal "2_470_000mutez")
+            ~adjustment_index:fixedpoint_one
+            ~collateral_at_auction:(Ligo.tez_from_literal "3_000_000mutez")
+            ~last_touched:(Ligo.timestamp_from_seconds_literal 0) in
+
+        assert_bool
+          "burrow_is_liquidatable returned true, but the burrow is expected to be non-liquidatable"
+          (not (Burrow.burrow_is_liquidatable Parameters.initial_parameters burrow))
+    );
```

Comment:
Ha, I'm very surprised we need those two to be honest! Given how many times `burrow_is_liquidatable` is called in `tests/testLiquidation.ml` I'd have thought that it's pretty well-tested. Having more tests is always good I guess, if they cover previously uncovered territory :slightly_smiling_face:

Reply:
I was as well! If I remember correctly, increasing the tez literal in this function (which corresponds to the tez scaling factor) _did_ get caught by our existing tests, but they didn't detect when it was set to a smaller value.